plugins/workspace-plugin/src/askpass-main.ts

```diff
@@ -1,4 +1,4 @@
-// tslint:disable:file-header
+/* eslint-disable header/header */
```

Comment:
To leave an origin license header it is enough to provide this type of directive? Just curious.

Reply:
yes it is.
Also here I just renamed the `tslint` rule to `eslint` rule.

Reply:
Got it.